### PR TITLE
chore: add some presentations

### PR DESCRIPTION
**Pull request by henryiii**
Adding a few talks and papers. @eduardo-rodrigues, if this is the link you want for PyHEP 2020, it should be pretty easy to collect the other talks from there.

### CODE REVIEW

File: pages/resources.md [hunk 1]
```diff
@@ -16,24 +16,34 @@ to, the Scikit-HEP project.
...
+    [arXiv:1905.00002 \[physics.comp-ph\]](https://arxiv.org/abs/1905.00002)
+-   *Bayesian Block Histogramming for High Energy Physics*,
+    Brian Pollack, Saptaparna Bhattacharya, Michael Schmitt,
+    [arXiv:1708.00810 \[physics.data-an\]](https://arxiv.org/abs/1708.00810)
 
 
 Presentations
 -------------
 
+-   **13-17 July 2020:** [PyHEP 2020 Workshop](https://indico.cern.ch/event/882824/overview), Virtual:
```
Comment (eduardo-rodrigues) [inline pages/resources.md:44]:
```suggestion
-   **13-17 July 2020:** [PyHEP 2020 Workshop](https://indico.cern.ch/e/PyHEP2020), virtual:
```

File: pages/resources.md [hunk 2]
```diff
@@ -16,24 +16,34 @@ to, the Scikit-HEP project.
 Proceedings and publications
 ----------------------------
 
--   *The Scikit HEP Project - overview and prospects*, Eduardo Rodrigues et al.,
+-   *Recent developments in histogram libraries*,
+    H. Dembinski, J. Pivarski and H. Schreiner,
+    proceedings of the [CHEP 2019 Conference](https://chep2019.org/), Adelaide Australia; 
+    [EPJ Web of Conferances 245, 05014 (2020)](https://doi.org/10.1051/epjconf/202024505014);
```
Comment (eduardo-rodrigues) [inline pages/resources.md:22]:
```suggestion
    [EPJ Web of Conferences 245, 05014 (2020)](https://doi.org/10.1051/epjconf/202024505014);
```

File: pages/resources.md (hunk 1 above)
Comment (henryiii) [reply pages/resources.md:44]: I was wondering if there was a form like this for this years PyHEP, there was last year.

Comment (eduardo-rodrigues) [reply pages/resources.md:44]: I try to have one since nicer :-).

**Comment by eduardo-rodrigues** [comment]
I meant to ask: what about the other presentations at for example https://github.com/scikit-hep/boost-histogram/blob/develop/README.md (bottom of the page)? You may also have other ones listed in your other package repositories?

**Pull request merged**